scripts/update-docs-index.ts

```diff
@@ -39,15 +39,12 @@ ${rules.map(toTableRow).join('\n')}
 `
 }
 
-const filePath = resolve(__dirname, '../docs/rules/README.md')
+const filePath = resolve(__dirname, '../docs/rules.md')
```

Comment:
I think we should probably change it.

```suggestion
const filePath = resolve(__dirname, '../docs/rules/index.md')
```